modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/taur_types.dm

```diff
@@ -7,27 +7,98 @@
 	center = TRUE
 	relevent_layers = list(BODY_ADJ_LAYER, BODY_FRONT_UNDER_CLOTHES)
 	genetic = TRUE
-	var/taur_mode = NONE //Must be a single specific tauric suit variation bitflag. Don't do FLAG_1|FLAG_2
-	var/alt_taur_mode = NONE //Same as above.
-	var/hide_legs = TRUE
-	var/hide_markings = FALSE //Any taur part that has "legs" should not hide markings
-
-/datum/sprite_accessory/taur/is_hidden(mob/living/carbon/human/H, obj/item/bodypart/HD)
-	if(H.wear_suit && (H.wear_suit.flags_inv & HIDEJUMPSUIT))
+	organ_type = /obj/item/organ/external/taur_body
+	flags_for_organ = SPRITE_ACCESSORY_HIDE_SHOES
+	/// Must be a single specific tauric suit variation bitflag. Don't do FLAG_1|FLAG_2
+	var/taur_mode = NONE
+	/// Must be a single specific tauric suit variation bitflag. Don't do FLAG_1|FLAG_2
```

Comment:
what happens if you do? Should test for it if it'll break things

Reply:
I just moved around the variable and made the comment into AUTODOC.

It doesn't really break anything for now, that I know of anyway, because it's always just a matter of checking which taur archetype your sprite belongs to. It's done as a bitflag to make it easier to be compared against.